llm/ext_server/server.cpp

```diff
@@ -2676,6 +2684,9 @@ static void server_params_parse(int argc, char **argv, server_params &sparams,
             exit(1);
         }
     }
+
+    gpt_params_handle_model_default(params);
```

Comment:
I don't think this is required (let me know if it is)

Reply:
Yeah spot on, a remnant from a previous incarnation 🤣,

Thanks, cleaned up.

Reply:
@jmorganca still keen to get this PR in if possible?